content/discrete-math/concepts/congruences/congruences.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'Congruences'
+Description: 'Congruences refer to the relationship between two integers a and b when they have the same remainder after
+division by positive integer m that is greater than 1.'
+Subjects:
+  - 'Computer Science'
+  - 'Discrete Math'
+Tags:
+  - 'Arithmetic'
+  - 'Math'
+  - 'Logical'
+  - 'Logic'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Congruences** refer to the relationship between two integers, _a_ and _b_, that have the same remainder after division
+by positive integer, _m_, that is greater than 1.
+
+## _a_ divides _b_
```

Comment:
The preview for this section seems a bit off. Can we replace $$\neq$$ with an alternative symbol or text

Reply:
Also, it may be nice to preface these headers based on their meaning.

Can we also do the same for the bottom portion that goes into the properties? So create a h2 named something like "Congruence Properties" and have a short summary of what the different properties entail before going into linear congruences, reflexive, etc.. (also turn those into h3s)